ui/src/components/visit-us/VisitUsPage.js

```diff
@@ -0,0 +1,192 @@
+import {
+  AspectRatio,
+  Box,
+  Center,
+  Container,
+  Flex,
+  Heading,
+  Icon,
+  Link,
+  LinkBox,
+  LinkOverlay,
+  Spacer,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import { RiArrowRightLine, RiChat1Line } from "react-icons/ri";
+import Faq from "./Faq";
+
+const VisitUsPage = (props) => {
+  return (
+    <Container maxW="container.lg" py={10}>
+      <VStack spacing={[4, 10]} align="stretch">
+        <Box
+          borderWidth="1px"
+          borderRadius="20"
+          bgImage={`url('${process.env.PUBLIC_URL}/images/visitus/header-banner.jpg')`}
+          bgPosition="center"
+          bgSize="cover"
+          px={[6, 12, 36]}
+          py={[8, 16, 20]}
+          mb={[4, 0]}
+        >
+          <Heading
```

Comment:
may have to change fontSize to a slightly smaller one for iphone 5/SE

![Uploading Screenshot 2021-08-18 at 8.53.45 AM.png…]()